source/python/gui/source/gui.py

```diff
@@ -187,29 +196,29 @@ def update_line_graph(sortFilter, func_list, exp_list, data, numPoints, samples)
     return mask_all, causalLayout, samplesLayout
 
 
-def reset_Input_filters(max_points):
+def reset_Input_filters(workloads, max_points):
```

Comment:
```suggestion
def reset_input_filters(workloads, max_points):
```

You don't have to accept this suggestion but please fix this inconsistent naming convention.